ops-bedrock/entrypoint-l1.sh

```diff
@@ -14,9 +14,9 @@ WS_PORT="${WS_PORT:-8546}"
 
 if [ ! -d "$GETH_KEYSTORE_DIR" ]; then
 	echo "$GETH_KEYSTORE_DIR missing, running account import"
-	echo -n "pwd" > "$GETH_DATA_DIR"/password
-	echo -n "$BLOCK_SIGNER_PRIVATE_KEY" | sed 's/0x//' > "$GETH_DATA_DIR"/block-signer-key
-	geth account import \
+	printf "%s" "pwd" > "$GETH_DATA_DIR"/password
```

Comment:
Reviewers: Seems this was also present in the original version, and the fix seems right but let me know if there's some reason we actually want to write the string `"pwd"` instead of the directory

Reply:
Ok, I see now this is pwd as in password, not the directory, so reverting this change